src/lib/Components/Artist/ArtistInsights/MarketStats.tsx

```diff
@@ -147,8 +173,8 @@ export const MarketStatsQueryRenderer: React.FC<{
       variables={{ artistInternalID }}
       query={graphql`
         query MarketStatsQuery($artistInternalID: ID!) {
-          priceInsights(artistId: $artistInternalID) {
-            ...MarketStats_priceInsights
+          priceInsightsConnection: priceInsights(artistId: $artistInternalID, sort: DEMAND) {
```

Comment:
```suggestion
          priceInsightsConnection: priceInsights(artistId: $artistInternalID, sort: DEMAND_RANK_DESC) {
```

I think this is the correct sort based on the schema.

Reply:
thanks for fixing this